### PR TITLE
[bug]Test fix Selenium , cypress and lint

### PR DESCRIPTION
**Pull request by anmode**

### CODE REVIEW

File: webpack.config-frontend.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 const VueLoaderPlugin = require('vue-loader/lib/plugin');
-
+const crypto = require("crypto");
```
Comment (heduenas) [inline webpack.config-frontend.js:7]: why are these 3 lines necessary?

File: yarn.lock [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 "@babel/cli@^7.14":
   version "7.14.5"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.14.5.tgz#9551b194f02360729de6060785bbdcce52c69f0a"
+  resolved "https://registry.npmjs.org/@babel/cli/-/cli-7.14.5.tgz"
```
Comment (heduenas) [inline yarn.lock:7]: why do we need to change from yarnpkg.com to npmjs.org? is it possible to stay with yarnpkg?

**Review by heduenas — COMMENTED** [review]
Kudos @anmode for taking the initiative to fix this issue. 

I left a couple of comments to better understand the changes you are making. I'm also rerunning the checks because I saw that the lint failed.

**Comment by heduenas** [comment]
> 

Actually, you can see the lint errors here: https://github.com/omegaup/omegaup/actions/runs/4299209221/attempts/1 they seem to be legit. Please fix those. 

Along with the lint errors I see other warnings about node js version 12. I suspect there are other places in the code where we need to update the node js version?

**Comment by pabo99** [comment]
> > 
> 
> Actually, you can see the lint errors here: https://github.com/omegaup/omegaup/actions/runs/4299209221/attempts/1 they seem to be legit. Please fix those.
> 
> Along with the lint errors I see other warnings about node js version 12. I suspect there are other places in the code where we need to update the node js version?

One of the lint errors has been fixed in the PR https://github.com/omegaup/omegaup/pull/6913. It might be a good idea to apply all the changes in this PR.

File: webpack.config-frontend.js (hunk 1 above)
Comment (anmode) [reply webpack.config-frontend.js:7]: Oh actually i used this for debugging. as in the error you can see openssl library crypto was creating problem so. i just used this. ok ig it is not required thanks. I removed it.

File: yarn.lock (hunk 2 above)
Comment (anmode) [reply yarn.lock:7]: Sure. I'll try to revert this. Ig it should work.

**Comment by anmode** [comment]
@heduenas @pabo99 I got the exact issue. as nodejs 16 is not causing any issue with webpack version. I was able to replicate as i was using node18 version. what to now? any comments

**Comment by anmode** [comment]
For the lint error it is ok. I will fix in this pR only. Ig container had been stopped from administrator side

**Comment by anmode** [comment]
I tried do not to update version as when do it in my local other dependencies creating problem in building and i have to overwrite many things. I hope these lines could fix selenium and cypress test. Thanks!

File: webpack.config.js [hunk 3]
```diff
@@ -3,6 +3,9 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 const frontendConfig = require('./webpack.config-frontend.js');
+const crypto = require("crypto");
```
Comment (lhchavez) [inline webpack.config.js:6]: why are these lines required? what happens if they are removed?

Comment (anmode) [reply webpack.config.js:6]: if they are removed we will get this error:  There are other methods but i find this suitable. 
```
Run yarn build
yarn run v1.22.19
$ cross-env webpack --mode=production
node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at BulkUpdateDecorator.hashFactory (/home/runner/work/omegaup/omegaup/node_modules/webpack/lib/util/createHash.js:138:18)
    at BulkUpdateDecorator.update (/home/runner/work/omegaup/omegaup/node_modules/webpack/lib/util/createHash.js:44:50)
    at OriginalSource.updateHash (/home/runner/work/omegaup/omegaup/node_modules/webpack/node_modules/webpack-sources/lib/OriginalSource.js:104:8)
    at NormalModule._initBuildHash (/home/runner/work/omegaup/omegaup/node_modules/webpack/lib/NormalModule.js:735:17)
    at handleParseResult (/home/runner/work/omegaup/omegaup/node_modules/webpack/lib/NormalModule.js:799:10)
    at /home/runner/work/omegaup/omegaup/node_modules/webpack/lib/NormalModule.js:852:4
    at processResult (/home/runner/work/omegaup/omegaup/node_modules/webpack/lib/NormalModule.js:623:11)
    at /home/runner/work/omegaup/omegaup/node_modules/webpack/lib/NormalModule.js:674:5
    at /home/runner/work/omegaup/omegaup/node_modules/loader-runner/lib/LoaderRunner.js:404:3
    at iterateNormalLoaders (/home/runner/work/omegaup/omegaup/node_modules/loader-runner/lib/LoaderRunner.js:233:10)
    at Array.<anonymous> (/home/runner/work/omegaup/omegaup/node_modules/loader-runner/lib/LoaderRunner.js:224:4)
    at runCallbacks (/home/runner/work/omegaup/omegaup/node_modules/webpack/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:25:15)
    at /home/runner/work/omegaup/omegaup/node_modules/webpack/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:198:4
    at /home/runner/work/omegaup/omegaup/node_modules/graceful-fs/graceful-fs.js:123:16 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v18.14.1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.``

Comment (anmode) [reply webpack.config.js:6]: These are required as current webpack version that we are using is producing error with openssl library. they[Openssl] have removed this function so webpack is producing this error !

Comment (lhchavez) [reply webpack.config.js:6]: sounds good. can a comment be added for reference?

Comment (anmode) [reply webpack.config.js:6]: sure! I will add. Thanks

File: stuff/lib/logs.py [hunk 4]
```diff
@@ -70,13 +71,13 @@ def init(program: str, args: argparse.Namespace) -> None:
             log_handler: logging.Handler = logging.FileHandler(args.logfile)
         else:
             log_handler = logging.StreamHandler()
-        formatter = _CustomJsonFormatter()
+        formatter: _CustomJsonFormatter = _CustomJsonFormatter()
```
Comment (lhchavez) [inline stuff/lib/logs.py:74]: this is not complaining about the lack of a type annotation here, but rather at the declaration of the class. we can ignore it for now:

```suggestion
        formatter = _CustomJsonFormatter(). # type: ignore
```

File: stuff/lib/logs.py [hunk 5]
```diff
@@ -70,13 +71,13 @@ def init(program: str, args: argparse.Namespace) -> None:
             log_handler: logging.Handler = logging.FileHandler(args.logfile)
         else:
             log_handler = logging.StreamHandler()
-        formatter = _CustomJsonFormatter()
+        formatter = _CustomJsonFormatter(). # type: ignore
```
Comment (lhchavez) [inline stuff/lib/logs.py:74]:
```suggestion
        formatter = _CustomJsonFormatter() # type: ignore
```

File: stuff/lib/logs.py [hunk 6]
```diff
@@ -70,13 +71,13 @@ def init(program: str, args: argparse.Namespace) -> None:
             log_handler: logging.Handler = logging.FileHandler(args.logfile)
         else:
             log_handler = logging.StreamHandler()
-        formatter = _CustomJsonFormatter()
+        formatter = _CustomJsonFormatter() # type: ignore
```
Comment (lhchavez) [inline stuff/lib/logs.py:74]:
```suggestion
        formatter = _CustomJsonFormatter()  # type: ignore
```

**Comment by anmode** [comment]
I'm not able to replicate the lint errors in my local.

**Review by lhchavez — APPROVED** [review]
